### ISSUE TITLE
Socket Bonus bug

### ISSUE THREAD

**Issue by vandershep** [ccshiro/cc-buglist#1970]
Currently, if you equip an item with gems in order to achieve a socket bonus, and then switch the gems out for better ones (which would normally result in losing the socket bonus), the socket bonus still remains on your stat sheets (and persists through logging out and logging back in). I am not certain whether or not the persisting values are actually being applied to your character, only that they remain on the stat sheet. Below are screenshots providing an example of this bug:

Gemming for socket bonus resulting in 3 hit rating

http://i.imgur.com/UtIDut5.png

Swapping the orange and blue gems for reds, losing the socket bonus, but still retaining the 3 hit rating

http://i.imgur.com/yO0rFgB.png

**Comment by Jumpka** [comment]
Noone reported this so far? I guess it's combination of these reports:
https://github.com/ccshiro/cc-buglist/issues/1659
https://github.com/ccshiro/cc-buglist/issues/1921
https://github.com/ccshiro/cc-buglist/issues/1190

TLDR: You have to socket the right colors when wearing that item, then the socket bonus will stay forever, if you socket it while in your bags, it won't work.

**Comment by Doppelmoral** [comment]
Have you observed the behaviour of this bug when
1) repeating the procedure (will it keep adding ?)
2) zone in and out of a portal
3) log out and back in ?

Valuable information that may help narrowing down the cause.

**Comment by cabbscripts** [comment]
1) No, stat bonus will not keep stacking if you keep swapping gems. Once it applies the socket bonus to the item it remains. Taking the item on/off doesn't do anything either.

2) Neither portals nor relogging affect the item. The only way to remove the socket bonus after you get it is by taking the item off and then changing the gem. Bonus will only persist if you swap gems with the item equipped.
